components/prism-latte.js

```diff
@@ -0,0 +1,178 @@
+/* TODO
+	Add support for the n:syntax attribute macro
+	Add support for string interpolation using {$advancedSyntax} within macros
+*/
+
+(function(Prism) {
+
+	var string_interpolation = {
```

Comment:
This variable is only used once in `inside.string[1].inside.interpolation`, so why not inline it?